p4-16/psa/PSA.mdk

```diff
@@ -1497,6 +1499,59 @@ table.
 A PSA implementation must implement tables that have both a
 `psa_direct_counter` and `psa_direct_meter` property.
 
+### Table entry timeout notification {#sec-idle-timeout}
+
+PSA uses the `psa_idle_timeout` to enable a table implementation send
```

Comment:
typo s/send/to send/

Reply:
actually enable an implementation send notifications is also correct.

Reply:
my bad that's what I get for commenting on syntax as a non-native speaker :/